docs/development/core/public/kibana-plugin-public.appbase.md

```diff
@@ -4,26 +4,25 @@
 
 ## AppBase interface
 
-
 <b>Signature:</b>
 
 ```typescript
-export interface AppBase 
+export interface AppBase
 ```
 
 ## Properties
 
-|  Property | Type | Description |
-|  --- | --- | --- |
-|  [capabilities](./kibana-plugin-public.appbase.capabilities.md) | <code>Partial&lt;Capabilities&gt;</code> | Custom capabilities defined by the app. |
-|  [chromeless](./kibana-plugin-public.appbase.chromeless.md) | <code>boolean</code> | Hide the UI chrome when the application is mounted. Defaults to <code>false</code>. Takes precedence over chrome service visibility settings. |
-|  [euiIconType](./kibana-plugin-public.appbase.euiicontype.md) | <code>string</code> | A EUI iconType that will be used for the app's icon. This icon takes precendence over the <code>icon</code> property. |
-|  [icon](./kibana-plugin-public.appbase.icon.md) | <code>string</code> | A URL to an image file used as an icon. Used as a fallback if <code>euiIconType</code> is not provided. |
-|  [id](./kibana-plugin-public.appbase.id.md) | <code>string</code> | The unique identifier of the application |
-|  [navLinkStatus](./kibana-plugin-public.appbase.navlinkstatus.md) | <code>AppNavLinkStatus</code> | The initial status of the application's navLink. Defaulting to <code>visible</code> if <code>status</code> is <code>accessible</code> and <code>hidden</code> if status is <code>inaccessible</code> See [AppNavLinkStatus](./kibana-plugin-public.appnavlinkstatus.md) |
-|  [order](./kibana-plugin-public.appbase.order.md) | <code>number</code> | An ordinal used to sort nav links relative to one another for display. |
-|  [status](./kibana-plugin-public.appbase.status.md) | <code>AppStatus</code> | The initial status of the application. Defaulting to <code>accessible</code> |
-|  [title](./kibana-plugin-public.appbase.title.md) | <code>string</code> | The title of the application. |
-|  [tooltip](./kibana-plugin-public.appbase.tooltip.md) | <code>string</code> | A tooltip shown when hovering over app link. |
-|  [updater$](./kibana-plugin-public.appbase.updater_.md) | <code>Observable&lt;AppUpdater&gt;</code> | An [AppUpdater](./kibana-plugin-public.appupdater.md) observable that can be used to update the application [AppUpdatableFields](./kibana-plugin-public.appupdatablefields.md) at runtime. |
-
+| Property                                                         | Type                                      | Description                                                                                                                                                                                                                                                             |

```

Comment:
Editor auto-formatted these files when fixing a merge conflict and egenerating doesn't seem to fix it either. I'm thinking of just leaving this as is because it shouldn't really matter... Any opposed?

Reply:
It doesn't alter the final markdown rendering, soI think this is fine (we are trying as much as possible to manually format the columns width in core doc, but I don't think this is a requirement, can someone else from the team confirm?)

Reply:
> Editor auto-formatted these files when fixing a merge conflict and egenerating doesn't seem to fix it either. 

Our script doesn't update docs if an interface hasn't changed. You can enforce update with `yarn core:acceptApiChanges --docs` or you can keep it as-is. The script will update it when the `appbase` interface changes.